default/hypr/bindings/tiling.conf

```diff
@@ -60,3 +60,27 @@ bindd = SUPER, mouse_up, Scroll active workspace backward, workspace, e-1
 # Move/resize windows with mainMod + LMB/RMB and dragging
 bindmd = SUPER, mouse:272, Move window, movewindow
 bindmd = SUPER, mouse:273, Resize window, resizewindow
+
+
+# Toggle groups
+bindd = SUPER ALT, G, Toggle tabbed grouping, togglegroup
+bindd = SUPER ALT, U, Move the active window out of its group, moveoutofgroup
+
+# Join groups
+bindd = SUPER ALT, left, Move window to the group in left, moveintogroup, l
+bindd = SUPER ALT, right, Move window to the group in right, moveintogroup, r
+bindd = SUPER ALT, up, Move window to the group in top, moveintogroup, u
+bindd = SUPER ALT, down, Move window to the group in bottom, moveintogroup, d
```

Comment:
Oh, thanks for that. I was still missing this part. :-D

It would also be nice if we were able to set a workspace as grouped by default.